src/components/input/elastic-textarea.ts

```diff
@@ -9,6 +9,9 @@
  * @param textarea - the target `textarea` element
  */
 export const createElasticTextArea = (textarea: HTMLTextAreaElement) => {
+  // Progressively enhances experience
+  textarea.classList.add('is-elastic');
```

Comment:
Should the `destroy` method that's returned by this function remove this class?

Reply:
Yes, it should. Good catch!

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/2098/commits/a09fe661f395079a5ac56ba8df74d8fb63889c80